racetime/templates/racetime/race_livesplit.html

```diff
@@ -0,0 +1,49 @@
+{% extends 'racetime/base.html' %} {% load static %} {% block css %}
+<base target="_blank">
+<link rel="stylesheet" href="{% static 'racetime/style/race.css' %}"> {% endblock %} {% block js %} {{ js_vars|json_script:"race-vars" }}
+<script src="{% static 'racetime/script/race.js' %}"></script>
+{% endblock %} {% block title %} Race room ({{ race.slug }}) | {{ race.category.name }} | {% endblock %} {% block meta %}
+<meta property="og:title" content="{{ race.category.name }} ~ {{ race.goal_str }} ({{ race.state_info.verbose_value }})"> {% if race.is_preparing %}
+<meta name="description" content="Race room open, {% if race.state == 'open' %}anyone may join this race{% else %}by invitation only{% endif %}.">
+<meta property="og:description" content="Race room open, {% if race.state == 'open' %}anyone may join this race{% else %}by invitation only{% endif %}."> {% elif race.is_pending or race.is_in_progress %}
+<meta name="description" content="{{ race.entrants_count }} entrant{{ race.entrants_count|pluralize }} are currently racing.">
+<meta property="og:description" content="{{ race.entrants_count }} entrant{{ race.entrants_count|pluralize }} are currently racing."> {% elif race.state == 'finished' %}
+<meta name="description" content="{{ race.entrants_count }} entrant{{ race.entrants_count|pluralize }} finished this race in {{ race.timer_str }}.">
+<meta property="og:description" content="{{ race.entrants_count }} entrant{{ race.entrants_count|pluralize }} finished this race in {{ race.timer_str }}."> {% else %}
+<meta name="description" content="This race was cancelled.">
+<meta property="og:description" content="This race was cancelled."> {% endif %} {% if meta_image %}
+<meta property="og:image" content="{{ meta_image }}"> {% endif %} {% endblock %} {% block header %}{% endblock header %} {% block nocurrentrace %}{% endblock nocurrentrace %} {% block footer %}{% endblock footer %} {% block main %}
+<style>
+    .timer {
+        display: none !important;
+    }
+    
+    .go-spectate {
+        display: none !important;
+    }
+</style>
```

Comment:
This section is a bit of a mess, could you clear it up a little? I think the `<meta>` tags are unnecessary here.